spec/support/fake_analytics.rb

```diff
@@ -56,7 +57,7 @@ def track_event(event, original_attributes = {})
               ERROR
             end
 
-            check_recursive.call(val, [key])
+            check_recursive.call(val, current_keypath)
```

Comment:
@zachmargolis I think you may have written this originally, I discovered that the recursion wasn't really working correctly since it only carried forward the current key, so the full path was not being evaluated. I don't think that was the intent?

See example at https://github.com/18F/identity-idp/pull/9572/files#diff-c1a67109f26d414eda38265f14fa49122409c743f83f5ae7488c3a1c935e9ee3